db/migrate/20210128160503_add_judicial_result_offence_relationship.rb

```diff
@@ -0,0 +1,11 @@
+class AddJudicialResultOffenceRelationship < ActiveRecord::Migration[6.0]
```

Comment:
From our chat with Ahilan last week, it appears a judicial result is associated to an offence and not directly to a hearing, so the `hearing_id` foreign key is unnecessary.

Reply:
Ok, so it’s unfortunate, but we have to keep this association between judicial result and hearing because the JSON schema expects us to specify a hearing ID for a judicial result.